### DISCUSSION TITLE
Problem when using 2 target variables as compared to using 1 target variable

### DISCUSSION THREAD

**Discussion by prabhatbanjara** [neuralhydrology/neuralhydrology#257] [Q&A]
I am working with an LSTM model in two different versions. In the first version, I used reservoir storage as the input, and the model was quite stable. In the second version, I am using both reservoir storage and streamflow as target variables, but the model has become highly unstable. I am using the Nash–Sutcliffe Efficiency (NSE) as the loss function, and the training losses are fluctuating significantly. Any insights or suggestions to help address this issue would be greatly appreciated.

<img width="883" height="557" alt="image" src="https://github.com/user-attachments/assets/69a89852-4635-402a-b242-565224a9d184" />

<img width="959" height="541" alt="image" src="https://github.com/user-attachments/assets/90a47a9f-2045-4434-a85c-79e91cb7fceb" />

<img width="964" height="557" alt="image" src="https://github.com/user-attachments/assets/d431f600-8019-4073-aeae-e446824477f3" />

<img width="960" height="547" alt="image" src="https://github.com/user-attachments/assets/0271beec-2bdc-4013-9f92-4b585b967798" />

<img width="956" height="342" alt="image" src="https://github.com/user-attachments/assets/d6b8967e-1448-44ee-afc4-4fead852f3d7" />
